### PR TITLE
[FIX] Members list title is wrong for other languages

### LINKED ISSUE #881: [BUG] Members list title resource is wrong for other supported languages
The title is wrong while data is being loaded/downloaded. Title should not be visible at the very start until data is downloaded or should be replaced for proper resource before controller appears and data is downloaded.

simulator screen shot - iphone x - 2017-10-29 at 14 41 49

### PR DESCRIPTION
**Pull request by cardoso**
@RocketChat/ios

Closes #881

### CODE REVIEW

File: Rocket.Chat/Controllers/Chat/MembersListViewController.swift [hunk 1]
```diff
@@ -110,6 +110,8 @@ extension MembersListViewController {
         if let cell = membersTableView.dequeueReusableCell(withIdentifier: LoaderTableViewCell.identifier) as? LoaderTableViewCell {
             self.loaderCell = cell
         }
+
+        title = data.title
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Chat/MembersListViewController.swift]: Isn't the title look like `Members (0)` this way?

**Comment by cardoso** [comment]
I'm not sure that's what we want, @dennispost. @rafaelks?

**Comment by dennispost** [comment]
@cardoso I'm down with you. There should be always a title. I just wanna say that #881 means a different way. 

If we want to continue to work on this screen I think we need something like this [DBEmptyState](https://github.com/dbsystel/DBEmptyState).

**Comment by rafaelks** [comment]
@cardoso Yes, your PR looks good! I also liked @dennispost suggestion to improve our empty state on lists of messages, members, files, etc. But I don't think we need a library to do that right now, we could implement that by ourselves... what do you guys think?

**Comment by artrmz** [comment]
@rafaelks, agree, should be in other idzie. I would love to work on that once I have a little bit of free time.

@cardoso, thank you for your PR.

**Pull request merged**